lib/sandboxed_renderer/init.js

```diff
@@ -20,19 +21,29 @@ for (let prop of Object.keys(events.EventEmitter.prototype)) {
 }
 Object.setPrototypeOf(process, events.EventEmitter.prototype)
 
-const electron = require('electron')
-const fs = require('fs')
-const preloadModules = new Map([
-  ['child_process', require('child_process')],
+const remoteModules = new Set([
+  'child_process',
+  'fs',
+  'os',
+  'path'
+])
+
+const loadedModules = new Map([
   ['electron', electron],
   ['events', events],
-  ['fs', fs],
-  ['os', require('os')],
-  ['path', require('path')],
-  ['url', require('url')],
-  ['timers', require('timers')]
+  ['timers', require('timers')],
+  ['url', require('url')]
 ])
 
+const {
+  preloadSrc, preloadError, webContentsId, platform, execPath, env
+} = electron.ipcRenderer.sendSync('ELECTRON_BROWSER_SANDBOX_LOAD', preloadPath)
+
+Object.defineProperty(process, 'webContentsId', {
+  configurable: false,
```

Comment:
`false` is a default value of the `configurable` attribute, no reasons to explicitly set it.
If you want to be explicit then you should also set `writable: false` (which is also a default value).